contracts/feature/interface/IDrop.sol

```diff
@@ -1,14 +1,52 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-import "./ILazyMint.sol";
+import "./IClaimConditionMultiPhase.sol";
 
-interface IDrop is ILazyMint {
+interface IDrop is IClaimConditionMultiPhase {
```

Comment:
so a Drop conceptually is `claim` + `setClaimConditions` ?

I think it makes sense, though I wonder if `claim` without `lazyMint` is a valid use case? (it might be)